components/google_cloud/actions/get-object/get-object.mjs

```diff
@@ -0,0 +1,50 @@
+import googleCloud from "../../google_cloud.app.mjs";
+
+export default {
+  name: "Get An Object",
+  version: "0.0.1",
+  key: "google_cloud-get-object",
+  type: "action",
+  description: "Downloads an object from a GCS bucket",
```

Comment:
Could you please add a link to the docs here?

Reply:
done